app/frontend/package.json

```diff
@@ -42,8 +42,8 @@
       "simple-import-sort"
     ],
     "rules": {
-      "simple-import-sort/imports": "error",
-      "simple-import-sort/exports": "error",
+      "simple-import-sort/imports": "warn",
+      "simple-import-sort/exports": "warn",
```

Comment:
Changed these to `warn` as it's annoying they fail builds in dev when you know you'll go back and fix this later/before you commit. They still fail on CI so it's just as good as error on build time